### DISCUSSION TITLE
Example project that shows running fumadocs in a separate root layout under `/docs`

### DISCUSSION THREAD

**Discussion by yannxaver** [fuma-nama/fumadocs#860] [Q&A]
1. All the example projects I found assume that fumadocs is running as the primary layout provider under `/`. Instead I want to run fumadocs in a separate root layout under `/docs` path.

```
app
  (web)
    ...other things
    layout.tsx 
  (docs)
    fumadocs
    layout.tsx
```

https://nextjs.org/docs/app/building-your-application/routing/route-groups#creating-multiple-root-layouts

2. If the project is already using `@next/mdx` does `@next/mdx` become obsolete and can be fully replaced by `fumadocs-mdx/next`?

When I replaced `@next/mdx` with `fumadocs-mdx/next` in `nextjs.config.mjs` I got this error:

```
landing.mdx:34:1-74:2: Unexpected `FunctionDeclaration` in code: only import/exports are supported
```

**Comment by fuma-nama** [comment]
1. Just move the providers and layouts to your new root layout, same as how you handle it in App Router.

2. next/mdx works but need further configurations, Fumadocs MDX is encouraged. You probably have invalid syntax in index.mdx, I need more info to investigate

**Comment by Gitstar-OC** [comment]
Hey, I know a group of people that already doing this.

**Reply by Gitstar-OC** [reply]
https://github.com/dokploy/dokploy, in this repo under the apps folder dokploy team has developed docs under the docs section, app under app and website section. you can use this for help and configuration.
